_data/projects/tldr-pages.yml

```
@@ -15,8 +15,8 @@ tags:
 - shell
 - cli
 upforgrabs:
-  name: help wanted
-  link: https://github.com/tldr-pages/tldr/labels/help%20wanted
+  name: Hacktoberfest
```

This value is case-sensitive when looking for it using the GitHub API, so we need it to match:

```suggestion
  name: hacktoberfest
```